DIPs/dip10ss.md

```diff
@@ -0,0 +1,100 @@
+# Enum and Function Parameter Attributes
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | 10ss                                                            |
+| Review Count:   | NA                                                              |
+| Author(s):      | @skl131313                                                      |
+| Status:         | NA                                                              |
+
+
+## Abstract
+
+Allow additional meta information (attributes) to be attached with enums and functions parameters, including built-in attributes such as `deprecated`.
+
+### Links
+
+[Does it require a DIP?](http://forum.dlang.org/thread/cltyrthdxkkfvuxqasqw@forum.dlang.org)
```

Comment:
Event though this is the title of the thread, better use a more descriptive title here, e.g. "Preliminary discussion about Enum and Function Parameter Attributes on the NG"